extensions/wikia/ImageReview/PromoteImageReview.setup.php

```diff
@@ -30,6 +30,7 @@
 if( function_exists('extAddBatchTask') ) {
 	extAddBatchTask($dir . "PromoteImageReviewTask.php", "promoteimagereview", "PromoteImageReviewTask");
 }
+//Possibly this line can be removed (as the usage new Wikia\\Tasks\\Tasks\\PromoteImageReviewTask was removed)

```

Comment:
could you create a ticket to investigate that? or do it as part of this code change?